pkg/ceph/client/pool.go

```diff
@@ -127,15 +127,22 @@ func CreatePool(context *clusterd.Context, clusterName string, newPool CephStora
 		}
 	}
 
+	// ensure that the newly created pool gets an application tag (just reuse the pool name)
+	args = []string{"osd", "pool", "application", "enable", newPool.Name, newPool.Name}
```

Comment:
Is there a reason to use the pool name as the application name? If not, should you use `rbd` since it seems what ceph uses by default. http://docs.ceph.com/docs/master/rados/operations/pools/#associate-pool-to-application